test/helper/helper.go

```diff
@@ -316,3 +323,163 @@ func NewRangeSlice(start, end int) []int {
 	}
 	return slice
 }
+
+// setupRawMongoClient returns the raw mongo client.
+func setupRawMongoClient(databaseName string) (*gomongo.Client, error) {
+	conf := &mongo.Config{
+		ConnectionTimeout: "5s",
+		ConnectionURI:     "mongodb://localhost:27017",
+		YorkieDatabase:    databaseName,
+		PingTimeout:       "5s",
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), conf.ParseConnectionTimeout())
+	defer cancel()
+
+	client, err := gomongo.Connect(
+		ctx,
+		options.Client().
+			ApplyURI(conf.ConnectionURI).
+			SetRegistry(mongo.NewRegistryBuilder().Build()),
+	)
+	if err != nil {
+		return nil, fmt.Errorf("connect to mongo: %w", err)
+	}
+
+	pingTimeout := conf.ParsePingTimeout()
+	ctxPing, cancel := context.WithTimeout(ctx, pingTimeout)
+	defer cancel()
+
+	if err := client.Ping(ctxPing, readpref.Primary()); err != nil {
+		return nil, fmt.Errorf("ping mongo: %w", err)
+	}
+
+	logging.DefaultLogger().Infof("MongoDB connected, URI: %s, DB: %s", conf.ConnectionURI, conf.YorkieDatabase)
+
+	return client, nil
+}
+
+// CleanUpAllCollections removes all data in every collection.
+func CleanUpAllCollections(databaseName string) error {
+	cli, err := setupRawMongoClient(databaseName)
+	if err != nil {
+		return err
+	}
+
+	collections := []string{mongo.ColProjects, mongo.ColUsers, mongo.ColClients,
+		mongo.ColDocuments, mongo.ColChanges, mongo.ColSnapshots, mongo.ColSyncedSeqs}
```

Comment:
When we add a new collection, it seems easy to forget to put it here.
How about `mongo` package exposing this array?